x-pack/plugins/stack_connectors/public/connector_types/index.ts

```diff
@@ -69,6 +70,7 @@ export function registerConnectorTypes({
   connectorTypeRegistry.register(getTorqConnectorType());
   connectorTypeRegistry.register(getTinesConnectorType());
   connectorTypeRegistry.register(getD3SecurityConnectorType());
+  connectorTypeRegistry.register(getTheHiveConnectorType());
```

Comment:
Hey @cnasikas, does the new convention to put new connectors under a feature flag apply to this PR?

Reply:
Hey! Yes, we need to follow the new intermediate release process for all new connectors.